src/components/Header/index.tsx

```diff
@@ -16,6 +16,11 @@ const Header: React.FC = () => {
   const handleMaximize = useCallback(() => {
     const window = remote.getCurrentWindow()
 
+    const isMacSystem = os.platform() === 'darwin'
+    if (isMacSystem) {
+      return window.setFullScreen(!window.isFullScreen())
+    }
```

Comment:
I prefer using a else statement here instead of a return to block the code flow, what do you think?

Reply:
I usually return to leave the function as quickly.
It is like a book separated by paragraphs.
when using if... else..., I can easily string together several conditions that can make reading difficult.
What is your opinion about that?

Reply:
As we only have two ways here i don't think it's a bad solution to use `if/else`. I think it's more descriptive doing this:
```js
if (isMacOS) {
  // do something in mac
} else {
  // do something in others
}
```
As in the future someone can change stuff inside the `if` and accidentally remove the `return`.

Reply:
I can do it but I don’t agree very much
We can see the function itself and how it gets cleaner
If we return as quickly as possible, it will be cleaner.
```js
const handleMaximize = useCallback(() => {
    const window = remote.getCurrentWindow()

    const isMacSystem = os.platform() === 'darwin'
    if (isMacSystem) {
      return window.setFullScreen(!window.isFullScreen())
    }

    const { width: currentWidth, height: currentHeight } = window.getBounds()
    const { width: maxWidth, height: maxHeight } = remote.screen.getPrimaryDisplay().workAreaSize
    const isMaximized = (currentWidth === maxWidth && currentHeight === maxHeight)

    if (!isMaximized) {
      return window.maximize()
    } 
     return window.unmaximize()
  }, [])
```

```js
const handleMaximize = useCallback(() => {
    const window = remote.getCurrentWindow()

    const isMacSystem = os.platform() === 'darwin'

    if (isMacSystem) {
      return window.setFullScreen(!window.isFullScreen())
    } else {
       const { width: currentWidth, height: currentHeight } = window.getBounds()
       const { width: maxWidth, height: maxHeight } = remote.screen.getPrimaryDisplay().workAreaSize
       const isMaximized = (currentWidth === maxWidth && currentHeight === maxHeight)

       if (!isMaximized) {
         window.maximize()
       } else {
         window.unmaximize()
      }
    }
  }, [])
```

Reply:
The problem is not with witch one is the cleaner function, but if it's descriptive itself. I think return shouldn't be used in this context to tell that the execution has stopped there. But, no problem, i'll merge the PR for now as this is not important at all for now! Thanks a lot for the contribution :purple_heart: